docs/howtos/airgap/03-hauler.md

```diff
@@ -0,0 +1,199 @@
+---
+sidebar_label: Hauler installation
+title: Air gap installation with Hauler
+description: Air gap installation for Kubewarden using Hauler
+keywords: [kubewarden, kubernetes, air gap installation, hauler]
+doc-persona: [kubewarden-operator, kubewarden-integrator]
+doc-type: [howto]
+doc-topic: [operator-manual, air gap, installation, hauler]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/howtos/airgap/hauler-sync"/>
+</head>
+
+This guide shows you how to install Kubewarden in air-gapped environments using
+[Hauler](https://docs.hauler.dev/docs/intro). Hauler is a tool that helps users
+run workloads in air-gapped environments. It moves the required resources for
+apps into those air-gapped environments.
+
+Kubewarden uses Hauler capabilities by providing a manifest file with all the
+required resources to run Kubewarden in a private environment. This
+documentation describes how you can use it. Refer to the Hauler documentation
+to learn more about it.
+
+The basic workflow using the Hauler manifest is:
+
+1. Get it from a Kubewarden release page
+2. Load all the resources required to run Kubewarden into a local store
+3. Export them into a file
+4. Move the file into the private environment
+5. Load the resources into Hauler running inside the private environment
+6. Copy everything into a registry, to be used in the installation process, in
+   your air-gapped environment
+
+## Download Hauler manifest from Helm chart release page
```

Comment:
Just realized, can we make the sections numbered as the numbered list?

Reply:
I'm not sure. The section are more a logic separation of the text. The goal was not follow the numbered list. Can we keep it as it is now?